packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -4310,6 +4310,27 @@ void main() {
 
     expect(tester.takeException(), isNull);
     expect(tester.getSize(findMenuItemButton(menuChildren.first.label)).width, 150.0);
+
+    // The overwrite of menuStyle is different when a width is provided but maximumSize is not,
+    // So it needs to be tested separately.
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          body: DropdownMenu<TestMenu>(
+            width: 200.0,
+            dropdownMenuEntries: menuChildren,
+            menuStyle: const MenuStyle(),
+          ),
+        ),
+      ),
+    );
+
+    await tester.tap(find.byType(TextField));
+    await tester.pumpAndSettle();
+
+    expect(tester.takeException(), isNull);
+    expect(tester.getSize(findMenuItemButton(menuChildren.first.label)).width, 200.0);
+
```

Comment:
Linux also shows a format error, removing this empty line should fix it.
```suggestion
```

Reply:
I tried this test locally and looks like it can pass without the PR change. It is because it's updating the existing test and previously, the menu hasn't been closed so the width is probably still the value from the code above. Can we create a new test for this part of testing?

Reply:
Any updates here:)?